src/test/javascript/controllers/urlInputTest.js

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Google LLC
```

Comment:
Can you pass this file through a linter so that these ESLint warnings are removed? Since this is a new file, we should ensure that it starts out in a good state.

There's some instructions on how to do that here:
https://eslint.org/docs/user-guide/command-line-interface